pageserve/app.py

```diff
@@ -151,22 +198,29 @@ def parse_events(events_dict):
     return events_dict['events']
 
 
-def get_user_info(user, url):
-    """Gets info about the current user from the users service."""
-    r = requests.post(url, data={'user_id': user})
-    response = r.json()
-    return response
-
-
-def has_edit_access(user_data):
+def has_edit_access(user):
     """Determines if the user with the given info has edit access."""
-    return user_data['edit_access']
+    if user is None:
+        return False
+    url = app.config["USERS_ENDPOINT"] + "authorization"
+    response = requests.post(url, data={"user_id": user["user_id"]})
+    return bool(response.json()["edit_access"]) if user else False
```

Comment:
what's the value of 'edit_access'? do we want to do a comparison rather than cast to bool?

Reply:
`response.json()` confusingly returns a Python dict, so `["edit_access"]` will be a bool. The `if user` check is redundant from above and casting to bool might be a little unsafe, so I just compared to `True` in https://github.com/knative-portability/large-events/pull/37/commits/0ff93b6261cef48ed24ed0082ab6793027e6031a